base/utils/R/convert.input.R

```diff
@@ -523,8 +525,12 @@ convert.input <- function(input.id, outfolder, formatname, mimetype, site.id, st
     
     if (forecast && !is.null(input.id) && !is.na(input.id)) { # for met2model coversion, arguments will be extraneous otherwise.
       fcn.args$year.fragment = TRUE
-      parent.inputfile.name <- PEcAn.DB::db.query(paste0("SELECT name FROM inputs WHERE id =", input.id), con)
-      fcn.args$in.data.file = parent.inputfile.name$name #Sends the file name (minus the extension)
+      if (grepl("NOAA_GEFS\\.[0-9]+", fcn.args$in.prefix)) { # Protects legacy database entries.
```

Comment:
Given that the 'legacy' is just a few weeks, I would go fix the legacy database entries rather than introduce a permanent hack into convert.input